code/modules/mob/living/simple_animal/friendly/mule_bot/mule_bot_modules.dm

```diff
@@ -0,0 +1,182 @@
+//towing moduke -- uncertain
+//pepperball module -- might make
+//motion detector module -- will make
+//auto flare mod -- finished
+//medical mod -- uncertain
+//personal storage module -- finished
+//storage module -- finished
+
+/obj/item/mule_module
+	name = "mule module"
+	desc = "Can be used to improve a mule and specialize it"
+	icon = 'icons/obj/items/items.dmi'
+	icon_state = "modkit"
+	var/mob/living/simple_animal/mule_bot/attached_mule
+	var/mutable_appearance/mod_overlay
+	var/overlay_icon = 'icons/mob/kerfus.dmi'
```

Comment:
idk why this var exists, you never change it

Reply:
I have yet to make the actually overlays for the mods.